extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/stream/YoutubeStreamExtractorDefaultTest.java

```diff
@@ -464,4 +466,35 @@ void testGetLicence() throws ParsingException {
             assertEquals("Creative Commons Attribution licence (reuse allowed)", extractor.getLicence());
         }
     }
+
+    public static class AudioTrackLanguage {
+
+        private static final String ID = "kX3nB4PpJko";
+        private static final String URL = BASE_URL + ID;
+        private static StreamExtractor extractor;
+
+        @BeforeAll
+        public static void setUp() throws Exception {
+            YoutubeTestsUtils.ensureStateless();
+            NewPipe.init(DownloaderFactory.getDownloader(RESOURCE_PATH + "audioTrack"));
+            extractor = YouTube.getStreamExtractor(URL);
+            extractor.fetchPage();
+        }
+
+        @Test
+        void testCheckAudioStreams() throws Exception {
+            assertTrue(extractor.getAudioStreams().size() > 0);
+
+            for (final AudioStream audioStream : extractor.getAudioStreams()) {
+                assertNotNull(audioStream.getAudioTrackName());
+            }
+
+            assertTrue(
+                    extractor.getAudioStreams()
+                            .stream()
+                            .anyMatch(audioStream -> audioStream.getAudioTrackName().equals("English"))
+            );
+        }
```

Comment:
I think we should also test a stream with multiple audio tracks and check that at least two known tracks are extracted correctly.